source/plot/ScatterPlotData2D.ooc

```diff
@@ -29,12 +29,12 @@ ScatterPlotData2D: class extends PlotData2D {
 	init: func ~default {
 		super()
 	}
-	init: func ~dataSeries (dataSeries: VectorList<FloatPoint2D>, label := "", colorRgba := ColorRgba new(), size := 5.f, shape := Shape Circle) {
+	init: func ~dataSeries (dataSeries: VectorList<FloatPoint2D>, label := "", colorRgba := ColorRgba transparent, size := 5.f, shape := Shape Circle) {
```

Comment:
This might not be what we wanted but that is how the tests are written now. Changes in behaviour should go in separate PR.